packages/host/app/resources/file.ts

```diff
@@ -100,23 +100,27 @@ class _FileResource extends Resource<Args> {
 
   private read = restartableTask(async () => {
     let response = await this.loaderService.loader.fetch(this._url, {
-      headers: {
-        Accept: 'application/vnd.card+source',
-      },
+      headers: { Accept: SupportedMimeType.CardSource },
     });
 
     if (!response.ok) {
-      log.error(
-        `Could not get file ${this._url}, status ${response.status}: ${
-          response.statusText
-        } - ${await response.text()}`,
-      );
-      if (response.status === 404) {
-        this.updateState({ state: 'not-found', url: this._url });
-      } else {
-        this.updateState({ state: 'server-error', url: this._url });
+      response = await this.loaderService.loader.fetch(this._url, {
+        headers: { Accept: SupportedMimeType.CardJson },
+      });
```

Comment:
This fallback helps find instances even if the url sent in does not end with `.json`